pkg/trace/stats/aggregation.go

```diff
@@ -99,6 +100,13 @@ func NewAggregationFromSpan(s *StatSpan, origin string, aggKey PayloadAggregatio
 	return agg
 }
 
+func processTagsHash(processTags string) uint64 {
+	if processTags == "" {
+		return 0
+	}
+	return peerTagsHash(strings.Split(processTags, ","))
```

Comment:
nit: can we rename `peerTagsHash` to be more general now that we're using it in both places?